### ISSUE TITLE
[a11y] Newsletter complementary landmark should be top level (/about/manifesto/)

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15336]
# Axe-core® Accessibility Results for /en-US/about/manifesto/

Page URL: https://www-dev.allizom.org/en-US/about/manifesto/

## axe-core found 1 violation

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | Aside should not be contained in another landmark | landmark-complementary-is-top-level | Best practice | moderate | 1   |

### Failed

1. Aside should not be contained in another landmark

[Learn more](https://dequeuniversity.com/rules/axe/4.10/landmark-complementary-is-top-level?application=playwright)

landmark-complementary-is-top-level

Best practice

Ensure the complementary landmark or aside is at top level

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.mzp-c-newsletter<br>```<br><br>**Element source**<br><br>```<br><aside class="mzp-c-newsletter"><br>``` | Fix any of the following:<br><br>*   The null landmark is contained in another landmark. |

**Comment by alexgibson** [comment]
Tip: a suitable fix for this would be to move the newsletter section of the page out of the top level `<main>` landmark, so that the `<aside>` (and its `mzp-l-content` parent element) come immediately after `<main>`.

**Comment by clarapenn** [comment]
Hi, I'd like to work on this one - thanks a lot!
